packages/core/src/App/Containers/CitizenshipModal/same-citizenship-modal.jsx

```diff
@@ -12,7 +12,7 @@ const SameCitizenshipModal = ({
 }) => (
     <div>
         <Text as='p' className='account-signup__text'>
-            {localize('Are you a citizen of the same country where you live?')}
+            {localize('Are you a citizen of {{- residence}}?', { residence })}
```

Comment:
@suisin-deriv Just curious, What does `-` do? 🤔 Couldn’t find anything on i18next doc.

Reply:
you can check on [this documentation](https://www.i18next.com/translation-function/interpolation) @farzin-fs

Reply:
@suisin-deriv Thank you 🙇🏻